### ISSUE TITLE
Plaidml-v1 & performance gain

### ISSUE THREAD

**Issue by stephanraaijmakers** [plaidml/plaidml#915]
I don't observe performance gains for an LSTM with plaidml-keras 0.7.0 and PLAIDML_USE_STRIPE=1. Does this work with plaidml==0.6.4?

**Comment by dgkutnic** [comment]
As per the recent discussion on #880, we're aware that LSTMs are a lagging performance area for PlaidML, and we're working on fixing it, especially in PlaidML-v1. Some of the changes needed to optimize LSTMs have been made much more feasible by our migration to MLIR and infrastructure changes made in PlaidML-v1.

You won't see these changes in the 0.7.0 release because that's a release based on a fixed point in time where improvements to LSTMs had not yet been made. As we keep working towards a production-ready PlaidML-v1, we'll keep you updated on any performance progress we see internally on LSTMs.

**Comment by Gerzer** [comment]
Does this mean that the current “v1” branch has LSTM performance improvements already?

**Comment by dgkutnic** [comment]
No, that's not what I was trying to say. I am just saying that any future performance improvements we make on LSTMs would be seen in `plaidml-v1`.

**Comment by Gerzer** [comment]
Thanks for clarifying. I love what you’re doing with PlaidML, and I look forward to its future!

**Comment by dgkutnic** [comment]
Thanks for your continued support!
